questionnaires/eq5d5l/eq5d5l_armt.json

```diff
@@ -0,0 +1,212 @@
+[
+    {
+        "field_name": "eq5d5l_1",
+        "form_name": "eq5d5l",
+        "section_header": "",
+        "field_type": "radio",
+        "field_label": "Please select the ONE option that best describes your health TODAY. Mobility:",
```

Comment:
Is there some other way of highlighting imp bits other than just capitalising?

Reply:
I don't think we currently support another way since we don't accept HTML in the field labels or section headers, only in the descriptive input content.